src/pages/host/HostPage.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { Redirect, Route, Switch, NavLink as RRNavLink } from 'react-router-dom';
+import { Listing, CREATE_LISTING, GET_HOST_LISTINGS } from 'networking/listings';
+import { Container, Fade, Button, NavItem, NavLink, Nav, Row, Col } from 'reactstrap';
+import AuthenticatedRoute from 'HOCs/AuthenticatedRoute';
+import NotFound from '../notFound';
+import { compose, graphql } from 'react-apollo';
+import { getFriendlyErrorMessage } from 'utils/validators';
+
+const HostRoutes = [
```

Comment:
Small point, but the naming here is inconsistent; UpperCamelCase makes this look like either a class or a React component, given the conventions we use elsewhere. I think `HOST_ROUTES` would be most consistent with the way we've named other top-level constants.

Reply:
kk thanks! I was confusing the use of UPPERCASE with enums, which are in a sense "constants" themselves.